data/sql/updates/pending_db_world/rev_1612957477367642592.sql

```diff
@@ -0,0 +1,39 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1612957477367642592');
+
+-- Lower respawn of and add more spawn locations for Tirisfal Pumpkin
+
+DELETE FROM `gameobject` WHERE `id`=375 AND `guid` IN (45042, 45043, 45157, 45194, 45195, 45196, 45197, 45198, 45200, 45201);
+
+INSERT INTO `gameobject` (`id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
```

Comment:
You are still missing GUID.

You are inserting 31 rows (Which are missing GUID) and deleting 10 GUIDS that are not being inserted here?

Reply:
OK, then I missed the point. I delete the old ones and let the database itself handle what numbers it wanted to assign since it's auto-incremented.

Should I instead reuse the first 10 GUID's and then add the later spawns with higher GUIDs?

Reply:
This has been scrapped, so closing!